### PR TITLE
refactor(permission-ui): refactor permission-ui & user part can be customized

### PR DESCRIPTION
**Pull request by ybzky**
close #xxx

## Pull Request Checklist

- [ ] Related tickets or issues have been linked in the PR description (or missing issue).
- [ ] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [ ] Unit tests have been added for the changes (if applicable).
- [ ] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

**Review by Gggpound — APPROVED** [review]

File: packages/core/src/services/command/command.service.ts [hunk 1]
```diff
@@ -373,12 +373,11 @@ export class CommandService extends Disposable implements ICommandService {
 
             throw new Error(`[CommandService]: command "${id}" is not registered.`);
         } catch (error) {
-            this._logService.error(error);
-
             if (error instanceof CustomCommandExecutionError) {
                 // If need custom logic, can add it here
                 return false as R;
             } else {
+                this._logService.error(error);
```
Comment (wzhudev) [inline packages/core/src/services/command/command.service.ts:380]: Wouldn't the error get logged twice here?

File: packages/sheets-ui/src/commands/commands/worksheet-protection.command.ts [hunk 2]
```diff
@@ -37,6 +36,7 @@ export interface IDeleteWorksheetProtectionParams {
 export interface ISetWorksheetProtectionParams {
     permissionId: string;
     rule: IWorksheetProtectionRule;
+    oldRule: IWorksheetProtectionRule;
```
Comment (wzhudev) [inline packages/sheets-ui/src/commands/commands/worksheet-protection.command.ts:39]: Should have some JSDoc on command params.

**Review by wzhudev — CHANGES_REQUESTED** [review]
A few questions:

* How can users custom the users' list panel? If there a tutorial or something?
* There are two different commands with the same name `AddRangeProtectionCommand` and why?
* Can Node.js SDK users change permissions?

**Comment by wzhudev** [comment]
And I have to say that the demo on preview site looks really like a bug after you set some permissions and they do not appears in the sidebar.

**Comment by ybzky** [comment]
1. node supports setting permissions but has not started to be modified
2. the legacy code has been deleted
3. currently all changes to the panel are packaged in the univer-pro for QA verification because they involve request-related logic

**Review by wzhudev — APPROVED** [review]

**Pull request merged**